apps/email/js/cards/message_list.html

```diff
@@ -100,7 +100,8 @@ <h1 data-prop="headerNode" class="msg-listedit-header-label">
         <input data-event="focus:onSearchButton"
                class="msg-search-text-tease" type="text"
                dir="auto"
-               data-l10n-id="message-search-input" />
+               data-l10n-id="message-search-input"
+               data-prop="searchButton" />
```

Comment:
Since the issue is around the search box doing weird things before the rest of the JS code loads, since we are using an HTML cache to first show the page, the disabled attribute should be set in the HTML template.